src/main/java/hudson/plugins/throttleconcurrents/ThrottleQueueTaskDispatcher.java

```diff
@@ -139,6 +140,26 @@ public CauseOfBlockage canRun(Task task, ThrottleJobProperty tjp) {
                     return CauseOfBlockage.fromMessage(Messages._ThrottleQueueTaskDispatcher_MaxCapacityTotal(totalRunCount));
                 }
             }
+            
+            // check start interval between runs
+            if (tjp.getStartInterval().longValue() > 0) {        
+                if (task instanceof AbstractProject) {
+                    AbstractProject<?,?> p = (AbstractProject<?,?>) task;
+                    Run<?,?> lastBuild = p.getLastBuild();
+                    if (null != lastBuild && lastBuild.isBuilding()) {
```

Comment:
lastBuild is not helpful for concurrent runs, because there may be other running builds. :ant: